docs/en/getters.md

```diff
@@ -61,6 +61,27 @@ computed: {
 }
 ```
 
+You can also return functions in getters:
+```js
+getters: {
+  // ...
+  getTodoById: (state, getters) => (id) => {
+    return getters.todos.find(todo => todo.id === id)
+  }
+  // function syntax
+  getTodoById: function(state, getters){
```

Comment:
As this example has the same meaning as the above example, I'm still not sure why we need both the arrow function and normal function version in Vuex docs.